### ISSUE TITLE
Charset detection not default as utf-8

### ISSUE THREAD

**Issue by alexburley** [tomas/needle#306]
https://github.com/tomas/needle/issues/170

Basically a clone of this issue

https://www.w3.org/TR/encoding/ looks like utf-8 is now default?

Are you open to pull request?

**Comment by tomas** [comment]
Sure, go ahead!

**Comment by alexburley** [comment]
@tomas What's the testing situation. I've forked and set up tests but getting a lot of failures, is there any specific environment set up thats not in the Readme?

In particular

`decoder.spec.js -> test B -> encodes to UTF-8 `

Fails off the bat.

**Comment by tomas** [comment]
Yeah I just pushed a commit that breaks everything, haha. Give me five minutes.

**Comment by tomas** [comment]
Ok, go ahead and pull.

**Comment by alexburley** [comment]
@tomas wicked thanks

**Comment by alexburley** [comment]
@tomas Would you be okay with installing nock as a dev dependancy? Useful for specifying test cases for decoding. 

e.g.

```
const nock = require('nock');
const url = https://some.url.com
nock(url)
	.get('/')
	.reply(200, 'Magyarországi Fióktelepe', {
		'content-type': 'text/html; charset=EUC-JP',
	});

 needle.get(url, { decode: false }, function(err, resp) {
          ....
   })
```

**Comment by tomas** [comment]
Yeah, no prob.

**Comment by alexburley** [comment]
PR raised https://github.com/tomas/needle/pull/308

**Comment by alexburley** [comment]
@tomas How are releases managed? Is there a timeline for when this will be published?

**Comment by alexburley** [comment]
@tomas any update?
